spec/requests/api/merge_requests_spec.rb

```diff
@@ -168,39 +171,39 @@
 
       it "should return 400 when title is missing" do
         post api("/projects/#{project.id}/merge_requests", user),
-        target_branch: 'master', source_branch: 'stable'
+        target_branch: "master", source_branch: "stable"
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.